### PR TITLE
cloud: add Beta to metrics integration docs

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/third-party-monitoring-integrations.md [hunk 1]
```diff
@@ -27,21 +27,21 @@ The available third-party integrations are displayed.
 
 - Third-party metrics integrations are not available when the cluster status is **CREATING**, **RESTORING**, **PAUSED**, or **RESUMING**.
 
-## Available integrations
+## Available integrations
```
Comment (qiancai) [inline tidb-cloud/third-party-monitoring-integrations.md:30]:
```suggestion
## Available integrations
```

File: TOC-tidb-cloud.md [hunk 2]
```diff
@@ -248,13 +248,13 @@
   - [Airbyte](/tidb-cloud/integrate-tidbcloud-with-airbyte.md)
   - [Amazon AppFlow](/develop/dev-guide-aws-appflow-integration.md)
   - [Cloudflare](/tidb-cloud/integrate-tidbcloud-with-cloudflare.md)
-  - [Datadog](/tidb-cloud/monitor-datadog-integration.md)
+  - [Datadog (Beta)](/tidb-cloud/monitor-datadog-integration.md)
```
Comment (qiancai) [inline TOC-tidb-cloud.md:251]: 左侧目录这里可以不加 Beta 吗，看起来误以为是 Datadog 是 Beta

File: TOC-tidb-cloud.md [hunk 3]
```diff
@@ -248,13 +248,13 @@
...
   - [dbt](/tidb-cloud/integrate-tidbcloud-with-dbt.md)
   - [Gitpod](/develop/dev-guide-playground-gitpod.md)
   - [n8n](/tidb-cloud/integrate-tidbcloud-with-n8n.md)
   - [Netlify](/tidb-cloud/integrate-tidbcloud-with-netlify.md)
-  - [New Relic](/tidb-cloud/monitor-new-relic-integration.md)
-  - [Prometheus and Grafana](/tidb-cloud/monitor-prometheus-and-grafana-integration.md)
+  - [New Relic (Beta)](/tidb-cloud/monitor-new-relic-integration.md)
+  - [Prometheus and Grafana (Beta)](/tidb-cloud/monitor-prometheus-and-grafana-integration.md)
```
Comment (qiancai) [inline TOC-tidb-cloud.md:257]: ditto

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
-title: Integrate TiDB Cloud with Prometheus and Grafana
+title: Integrate TiDB Cloud with Prometheus and Grafana (Beta)
 summary: Learn how to monitor your TiDB cluster with the Prometheus and Grafana integration.
 ---
 
-# Integrate TiDB Cloud with Prometheus and Grafana
+# Integrate TiDB Cloud with Prometheus and Grafana (Beta)
 
-TiDB Cloud provides a [Prometheus](https://prometheus.io/) API endpoint. If you have a Prometheus service, you can monitor key metrics of TiDB Cloud from the endpoint easily.
+TiDB Cloud provides a [Prometheus](https://prometheus.io/) API endpoint (Beta). If you have a Prometheus service, you can monitor key metrics of TiDB Cloud from the endpoint easily.
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:8]:
```suggestion
TiDB Cloud provides a [Prometheus](https://prometheus.io/) API endpoint (beta). If you have a Prometheus service, you can monitor key metrics of TiDB Cloud from the endpoint easily.
```

File: tidb-cloud/monitor-new-relic-integration.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
 ---
-title: Integrate TiDB Cloud with New Relic
+title: Integrate TiDB Cloud with New Relic (Beta)
 summary: Learn how to monitor your TiDB cluster with the New Relic integration.
 ---
 
-# Integrate TiDB Cloud with New Relic
+# Integrate TiDB Cloud with New Relic (Beta)
 
 You can configure TiDB Cloud to send metric data of your TiDB clusters to [New Relic](https://newrelic.com/). After that, you can directly view these metrics in your New Relic dashboards.
```
Comment (qiancai) [inline tidb-cloud/monitor-new-relic-integration.md:8]:
```suggestion
TiDB Cloud supports New Relic integration (beta). You can configure TiDB Cloud to send metric data of your TiDB clusters to [New Relic](https://newrelic.com/). After that, you can directly view these metrics in your New Relic dashboards.
```

File: tidb-cloud/monitor-datadog-integration.md [hunk 6]
```diff
@@ -1,9 +1,9 @@
 ---
-title: Integrate TiDB Cloud with Datadog
+title: Integrate TiDB Cloud with Datadog (Beta)
 summary: Learn how to monitor your TiDB cluster with the Datadog integration.
 ---
 
-# Integrate TiDB Cloud with Datadog
+# Integrate TiDB Cloud with Datadog (Beta)
 
 You can configure TiDB Cloud to send metric data about your TiDB clusters to [Datadog](https://www.datadoghq.com/). After that, you can view these metrics in your Datadog dashboards directly.
```
Comment (qiancai) [inline tidb-cloud/monitor-datadog-integration.md:8]:
```suggestion
TiDB Cloud supports Datadog integration (beta). You can configure TiDB Cloud to send metric data about your TiDB clusters to [Datadog](https://www.datadoghq.com/). After that, you can view these metrics in your Datadog dashboards directly.
```

File: tidb-cloud/third-party-monitoring-integrations.md [hunk 7]
```diff
@@ -29,19 +29,19 @@ The available third-party integrations are displayed.
 
 ## Available integrations
 
-### Datadog integration
+### Datadog integration (Beta)
```
Comment (qiancai) [inline tidb-cloud/third-party-monitoring-integrations.md:32]:
```suggestion
### Datadog integration (beta)
```

File: tidb-cloud/third-party-monitoring-integrations.md [hunk 8]
```diff
@@ -29,19 +29,19 @@ The available third-party integrations are displayed.
...
 
 With the Datadog integration, you can configure TiDB Cloud to send metric data about your TiDB clusters to [Datadog](https://www.datadoghq.com/) and view these metrics in your Datadog dashboards.
 
 For the detailed integration steps and a list of metrics that Datadog tracks, refer to [Integrate TiDB Cloud with Datadog](/tidb-cloud/monitor-datadog-integration.md).
 
-### Prometheus and Grafana integration
+### Prometheus and Grafana integration (Beta)
```
Comment (qiancai) [inline tidb-cloud/third-party-monitoring-integrations.md:38]:
```suggestion
### Prometheus and Grafana integration (beta)
```

File: tidb-cloud/third-party-monitoring-integrations.md [hunk 9]
```diff
@@ -29,19 +29,19 @@ The available third-party integrations are displayed.
...
 
 With the Prometheus and Grafana integration, you can get a scrape_config file for Prometheus from TiDB Cloud and use the content from the file to configure Prometheus. You can view these metrics in your Grafana dashboards.
 
 For the detailed integration steps and a list of metrics that Prometheus tracks, see [Integrate TiDB Cloud with Prometheus and Grafana](/tidb-cloud/monitor-prometheus-and-grafana-integration.md).
 
-### New Relic integration
+### New Relic integration (Beta)
```
Comment (qiancai) [inline tidb-cloud/third-party-monitoring-integrations.md:44]:
```suggestion
### New Relic integration (beta)
```

File: tidb-cloud/third-party-monitoring-integrations.md [hunk 10]
```diff
@@ -1,11 +1,11 @@
...
-title: Third-Party Metrics Integrations
+title: Third-Party Metrics Integrations（Beta）
 summary: Learn how to use third-party metrics integrations.
 ---
 
-# Third-Party Metrics Integrations
+# Third-Party Metrics Integrations（Beta）
 
-You can integrate TiDB Cloud with third-party metrics services to receive TiDB Cloud alerts and view the performance metrics of your TiDB cluster using the metrics services.
+You can integrate TiDB Cloud with third-party metrics services (beta) to receive TiDB Cloud alerts and view the performance metrics of your TiDB cluster using the metrics services.
```
Comment (qiancai) [inline tidb-cloud/third-party-monitoring-integrations.md:8]:
```suggestion
You can integrate TiDB Cloud with third-party metrics services to receive TiDB Cloud alerts and view the performance metrics of your TiDB cluster using the metrics services. The third-party metrics integrations are currently in beta.
```

**Review by qiancai — APPROVED** [review]
/lgtm

**Review by hfxsd — APPROVED** [review]

**Pull request merged**